pkg/backend/display/progress.go

```diff
@@ -630,6 +625,19 @@ func (display *ProgressDisplay) refreshAllRowsIfInTerminal() {
 	}
 }
 
+func removeInfoColumnIfUnneeded(rows [][]string) {
+	// If there have been no info messages, then don't print out the info column header.
+	for i := 1; i < len(rows); i++ {
+		row := rows[i]
+		if row[len(row)-1] != "" {
```

Comment:
Are we assuming that `row` has some fixed length? Or at least that it has two or more elements?

Reply:
yes.  This is a grid.  So there number of rows is intended to always be the same.